### PR TITLE
Create included-example.md

### PR DESCRIPTION
**Pull request by an0d3va**
A brief description of what happens in the example included in the Zenon Dart SDK.

### CODE REVIEW

**Comment by georgezgeorgez** [comment]
This is an amazing first PR in the repo!
I'm using GitHub pages with built in Jekyll static site generator https://jekyllrb.com/docs/
Also the https://pmarsceill.github.io/just-the-docs/ theme
So I think for it to show up pretty on the zenon.wiki website, we might need to add a yaml header
Not sure if I can externalize it so people can just focus on the markdown, and I'll handle as much jekyll as possible
I need to setup a dev preview workflow for the site as well
Maybe for dev documentation, we don't really need it on the zenon.wiki site and can just direct people to github files since that shouldn't be an issue for devs

**Comment by georgezgeorgez** [comment]
expected output is nice

File: Dart-SDK-docs/included-example.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+## Imports 
+
+    import 'package:hex/hex.dart';
+    import 'package:znn_sdk_dart/znn_sdk_dart.dart';
+
+This example imports the whole SDK in one go, but it uses only functions from KeyStore and KeyPair.
+
+## Main part
+
+    This is the main routine of the example:
```
Comment (georgezgeorgez) [inline Dart-SDK-docs/included-example.md:18]: same comment as other pr
i don't think you meant to indent this

**Comment by an0d3va** [comment]
superseded by #4

**Pull request closed without merging**